moped-database/migrations/default/1748534889272_create_notes_view/up.sql

```diff
@@ -0,0 +1,631 @@
+-- Update moped_note_types to have source column
+ALTER TABLE moped_note_types
+ADD COLUMN source text NOT NULL DEFAULT 'moped';
+
+-- Insert eCapris status note type that will be used in notes view
+INSERT INTO moped_note_types (name, slug, source)
+VALUES
+('eCapris Status Update', 'ecapris_status_update', 'ecapris');
+
+COMMENT ON COLUMN moped_note_types.source IS 'Source of the note type, e.g., Moped or eCapris applications';
+
+-- Create the combined_project_notes view for the project notes tab and status update in summary view
+CREATE OR REPLACE VIEW combined_project_notes AS
+SELECT
+    ('M' || moped_proj_notes.project_note_id) AS id,
+    moped_proj_notes.project_note_id AS original_id,
+    moped_proj_notes.project_note,
+    moped_proj_notes.created_at,
+    moped_proj_notes.created_by_user_id,
+    moped_proj_notes.project_id,
+    (moped_users.first_name || ' ' || moped_users.last_name) AS author,
+    moped_note_types.name AS note_type_name,
+    moped_note_types.slug AS note_type_slug,
+    moped_phases.phase_name AS phase_name,
+    moped_phases.phase_key AS phase_key,
+    moped_proj_notes.is_deleted,
+    moped_proj_notes.phase_id,
+    TRUE AS is_editable,
```

Comment:
The intent of `is_editable` in this view is to help us know when to edit components like the pencil and trash icons. This would work along with the existing business logic that note authors can only edit their own notes and statuses.